### PR TITLE
Single node asg trimming

### PR DESCRIPTION
**Pull request by NorfairKing**

### CODE REVIEW

**Comment by ketzacoatl** [comment]
overall, looks good. A few nit-picky things:
* please keep whitespace fixups to their own commit
* it helps with readability (both in the GH UI as well as cli-based views of the git history), if the first line of the commit message is a concise / short summary, leaving sentences and longer explanations to the 3rd line and beyond. Most editors and git tooling help with that.
* I just updated `master` a little while ago, and am a fan of rebasing so git history is super readable (some consider me to be crazy, and I won't object).

none of these are blockers, just tweaks that help UX with git.

usually I leave the dev to merge as they see fit, so feel free to do so. You have my :+1:

**Comment by NorfairKing** [comment]
@ketzacoatl Thank you for the quick response.
Sorry about the whitespace. I have auto `terraform fmt` set up on saving, so I'll turn that off while I edit and re-do the commit and re-base. Will need to make sure to squash then, before merging.

**Comment by NorfairKing** [comment]
oh dear. That didn't work. Let me try again.

**Comment by NorfairKing** [comment]
There you go. I rebased ontop of master, squashed the changes and fixed the commit message. Is there anything left that you dislike?

**Comment by ketzacoatl** [comment]
looks great!
